### ISSUE TITLE
Add customization akin to Shimmer

### ISSUE THREAD

**Issue by Catomax262** [GiantLuigi4/TingedLights#2]
Well as they decided to abandon shimmer so

Honestly i'm a huge fan of colored light and i need to put it with other mods

**Comment by GiantLuigi4** [comment]
https://github.com/GiantLuigi4/TingedLights/tree/master/src/main/resources/assets/tingedlights

The colored lights can already be customized with resource packs

**Comment by Catomax262** [comment]
I don't think this is the proper way to do this, i can't stand having to manually open the texture of every block i want then open paint.net just to try to identify the color, copy-paste then edit the light values

Would be nice to kind of dump every light block to a file like Shimmer does

Or an eyedropper mode akin to Shimmer

**Comment by GiantLuigi4** [comment]
good point actually
